### ISSUE TITLE
Hyphenation data missing in non-English editions

### ISSUE THREAD

**Issue by platinorum** [tatuylonen/wiktextract#853]
There is no data for hyphenation in the output file.
This is possibly related to an old issue (#159).

Example:
["apple" on wiktionary](https://en.wiktionary.org/wiki/apple) vs. ["apple" on kaikki](https://kaikki.org/dictionary/English/meaning/a/ap/apple.html#en-apple-en-noun-en%3AQ89)

**Comment by kristian-clausal** [comment]
It's there. Open `raw data` and search for "hyphenation": it's present for the noun and the verb.

**Comment by xxyzz** [comment]
es edition has "syllabic" field in "sounds" lists, de edition's "Worttrennung" section currently is not extracted, fr and pl editions don't seem to have this kind of data.

**Comment by kristian-clausal** [comment]
Yeah, because the editions are each so different, data like `hyphenation` needs to be specially programmed into their respective extractors. In some languages, having separate fields for `hyphenation` makes no sense because they use predictable rules or syllables. Please keep in mind that English `hyphenation` data is only applicable to writing, specifically how you are supposed to divide words on line boundaries, it's not actual phonetic or 'real' language data.
